.github/workflows/check.yml

```
@@ -13,7 +13,7 @@ permissions: read-all
 jobs:
   audit:
     name: Audit
-    uses: ericcornelissen/svgo-action/.github/workflows/reusable-audit.yml@main
+    uses: ericcornelissen/svgo-action/.github/workflows/reusable-audit.yml@ci/add-gitleaks
```

This should be reverted before this Pull Request is merged

```suggestion
    uses: ericcornelissen/svgo-action/.github/workflows/reusable-audit.yml@main
```